history.rst

```diff
@@ -5,6 +5,35 @@
 OMERO version history
 =====================
 
+5.6.5 (June 2022)
+------------------
+
+This release includes the following upgrade of the OMERO.server Java components:
+
+- omero-gateway-java 5.6.10
+- omero-blitz 5.5.12
+- omero-server 5.6.4
+- omero-renderer 5.5.10
+- omero-romio 5.7.0
+- omero-common 5.5.10
+- omero-model 5.6.7
+
+
+Improvements include:
+
+-  a new server configuration allowing to control the pyramidal requirement for floating-point images
+-  an upgrade of Bio-Formats to version 6.10.0
+-  the inclusion of the experimental OMEZarrReader version 0.2.0 for reading OME-NGFF data
```

Comment:
This seems to introduce a problem when running
```
java -cp dist/lib/server/* `omero config get | awk '{print"-D"$1}'` ome.services.graphs.GraphPathReport /tmp/EveryObject.rst
Error: Could not find or load main class dist.lib.server.OMEZarrReader.jar
```

Reply:
Actually building using a non clean ``.m2`` leads to error see https://github.com/ome/openmicroscopy/pull/6324

Reply:
I think the failures are anterior to this PR but I agree they are currently blocking the proposed set of changes for OMERO.server 5.6.5 and should be resolved before moving forward.

Reply:
They are yes.